toml.md

```diff
@@ -454,7 +454,8 @@ Floats should be implemented as IEEE 754 binary64 values.
 A float consists of an integer part (which follows the same rules as decimal
 integer values) followed by a fractional part and/or an exponent part. If both a
 fractional part and exponent part are present, the fractional part must precede
-the exponent part.
+the exponent part. Omitting the integer part is invalid, as is omitting both
+of the fractional and exponent parts.
```

Comment:
I would drop this change. The new sentence just repeats what is already said clearly in the first sentence of the paragraph.

Reply:
I disagree. The fact that it isn't explicitly called out is what led me to file issue #759 in the first place.